apps/codeprop/Makefile.codeprop-tmp

```diff
@@ -0,0 +1,9 @@
+codeprop-tmp_src = codeprop-tmp.c
+
+# Enable large memory model supports for WISMOTE and EXP5438 platform
```

Comment:
As app makefiles are included before platform ones, it looks to me like we cannot check for CPU_HAS_MSP430X` here.

My only concern is the naming. `_HAS_` defines are usually defined by platforms, telling whether the platform has a given feature. A proposal would be to always keep `CPU_HAS_MSP430X` defined, and then set `TARGET_MEMORY_MODEL` to identify the memory model (and re-adjust the msp430 makefile)